cheatsheets/Java_Security_Cheat_Sheet.md

```diff
@@ -677,6 +685,8 @@ class AesGcmSimple {
 
 </details>
 
+_**End of code snippet.**_

```

Comment:
Agree with the previous reviewer, I think we can ditch this now that these are in collapsible sections.